casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -149,7 +147,7 @@ class TestNode[F[_]: Timer](
   val postGenesisStateHash = ProtoUtil.postStateHash(genesis)
 
   val shardConf = CasperShardConf(
-    -1,
+    0,
```

Comment:
Please use named arguments for literal values to prevent errors when order of parameters of the same type change and also it's easier to read the code.